applications/ClearDotTax.md

```diff
@@ -0,0 +1,257 @@
+- [polkadot-portfolio](https://github.com/substrate-portfolio/polkadot-portfolio) and [Polkaholic](https://twitter.com/polkaholic_io) fetch account balances from `frame-system`. However, it doesn't work for custom pallets and accrued rewards/fees from custom protocols.
+
+#### Are there any projects similar to yours in related ecosystems?
+
+- [CoinTracker](https://www.cointracker.io/)
+- [Koinly](https://app.koinly.io/)
+- [Netrunner](https://www.netrunner.tax/) - on Solana
+
+However, the above tools don't have support for Polkadot/Kusama and their Parachains (except EVM based ones).
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Team Lead: Rajdeep (https://twitter.com/rajgoesout)
+  - Building in the Polkadot ecosystem since ~2.5 years - led engineering at [YieldBay](https://github.com/yield-bay)
+  - Web3 Researcher/Rust Engineer
+  - Prev built products to solve for blockchain information asymmetry in the Filecoin ecosystem
+- Product Engineer: 0xDeep (https://twitter.com/0xDeep)
+  - Rust/Full-Stack Engineer
+  - Smart contract Auditor
+  - Prev wrote systems to process real-time data at a HFT firm
+- Product Designer: Axshay (https://twitter.com/axshay613)
+  - Designer [@polynomialfi](https://twitter.com/polynomialfi)
+  - Prev worked at a web3 game studio
+
+### Contact
+
+- **Contact Name:** Rajdeep Bharati
+- **Contact Email:** rajdeep@findsignal.studio
+- **Website:** https://www.findsignal.studio
+
+### Legal Structure
+
+- **Registered Address:** 3 Fraser Street, \#05-25 Duo Tower, Singapore (189352)
+- **Registered Legal Entity:** Find Signal Studio PTE. LTD.
+
+### Team's experience
+
+We've been building within the Polkadot ecosystem since 2020:
+
+- https://twitter.com/YieldScan - Staking dashboard for Polkadot and Kusama
+  - funded by Web3 Foundation Grants
+  - did 9 figures in transaction volume
+- https://twitter.com/yield_bay - Yield farming hub for Dotsama
+  - funded by the Polkadot Treasury
+  - tracks over 159 farms, 17 protocols, across 9 parachains
+
+### Team Code Repos
+
+- https://github.com/yield-bay/bay-watcher
+- https://github.com/yieldscan/yieldscan-frontend
+- https://github.com/yield-bay/bay-xcm-autocompound
+- https://github.com/yield-bay/bay-web
+
+**GitHub accounts of all team members:**
+
+- https://github.com/rajgoesout (yieldbay alt: https://github.com/JackSparrowYB)
+- https://github.com/0xdeepmehta
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/rajgoesout
+- https://twitter.com/0xDeep
+- https://twitter.com/axshay613
+
+### Project Details
+
+#### Key Deliverables
+
+- ##### Backend API
+  To fetch users' transactions across Polkadot and Kusama relay chains; and all major parachains of Polkadot & Kusama (ones supported by Subscan).
+- ##### [Transactions Dashboard](#milestone-1--transactions-dashboard)
```

Comment:
We usually expect some wireframes/designs for any frontend deliverables. Do you have anything you can share here?

Reply:
Got it. We're finalizing our designs - will attach them and ping on this thread as soon it's ready (should take about 3 days).